### ISSUE TITLE
CI / Test Coverage

### ISSUE THREAD

**Issue by mattxwang** [uclaacm/ACM-at-UCLA-Discord-Bot#29]
Given how large the codebase is already, I Imagine it might be tricky to evaluate new PRs and make sure that we don't introduce regressions. I'm not super familiar with Discord bot development, but I'm curious if we can write some tests for the commands / internal functionality and then run those on every commit/PR; that way, we can have some confidence that we won't break existing commands.

What I'm thinking of is something like a Jest setup for our functions, and then mocking database et al. instances, but someone with more experience should definitely chime in 😊

**Comment by Bwoltz** [comment]
I agree that adding tests should be priority number one. Especially if people from the dev team (like me) are going to want to jump in and add features. Unit testing the database-driven commands (like `!getStats`) shouldn't be too difficult if we can mock a database.

The only one that might be difficult to test is the `!iam` command since it relies on using the SendGrid API to send a verification code to the student's email.

I'm not the one who built this code base, so it might be beneficial to have one of the original contributors weigh in. But I will definitely look into this and hopefully come up with some tests. Thanks Matt.
